### DISCUSSION TITLE
第1节课笔记(11班)

### DISCUSSION THREAD

**Discussion by Shengshenlan** [InternLM/Tutorial#62] [笔记]
请在下方提交笔记链接

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/675918138

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/675924845

**Comment by BarryYin** [comment]
第一节课笔记：https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%B8%80%E8%8A%82.md

**Comment by chenyang126** [comment]
https://blog.csdn.net/weixin_50797351/article/details/135374993?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135374993%22%2C%22source%22%3A%22weixin_50797351%22%7D

**Comment by qianlian-mozi** [comment]
https://blog.csdn.net/m0_54824511/article/details/135375068?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135375068%22%2C%22source%22%3A%22m0_54824511%22%7D

**Comment by Fulin-Guo** [comment]
https://github.com/wdnmdm/Team_learning/blob/main/InternLM/%E7%AC%AC1%E8%8A%82%E8%AF%BE.md

**Comment by codefd** [comment]
https://juejin.cn/post/7319903348965457929

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/1.md

**Comment by zhanghui-china** [comment]
https://zhuanlan.zhihu.com/p/675952151

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135384017

**Comment by Everfighting** [comment]
https://github.com/Everfighting/InternLM_learning/blob/main/chapter1/chapter_1.md

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135384354

**Comment by limafang** [comment]
https://limafang.github.io/Azula_blogs.github.io/2024/01/04/Opmmlab%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98%E8%90%A5%E7%AC%94%E8%AE%B0.html

**Comment by Vacili-f** [comment]
https://zhuanlan.zhihu.com/p/676089337

**Comment by chenxiangzhen** [comment]
https://github.com/chenxiangzhen/InternLM-Tutorial/blob/main/%E7%AC%AC1%E8%8A%82.md

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/main/note/1/%E7%AC%AC%E4%B8%80%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by hellokid123** [comment]
https://github.com/hellokid123/internllm-notes/blob/main/%E7%AC%AC1%E8%AF%BE.md

**Comment by morning-start** [comment]
https://zhuanlan.zhihu.com/p/676119615

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135376847?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135376847%22%2C%22source%22%3A%22weixin_44986362%22%7D

**Comment by blackxer** [comment]
https://zhuanlan.zhihu.com/p/676142150

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/1%E3%80%81InternLM%E6%95%B4%E4%BD%93%E4%BB%8B%E7%BB%8D

**Comment by Unicornlyy** [comment]
http://t.csdnimg.cn/HxYA7

**Comment by zhilinfan** [comment]
https://zhuanlan.zhihu.com/p/676265258

**Comment by cc200723** [comment]
https://blog.csdn.net/weixin_46318072/article/details/135406639

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135410711?spm=1001.2014.3001.5501

**Comment by moyitech** [comment]
https://blog.csdn.net/qq_30650051/article/details/135416872

**Comment by Kongsea** [comment]
第一课笔记：https://blog.csdn.net/deeepsea/article/details/135417456

**Comment by wzl0329** [comment]
第一节课笔记：https://github.com/wzl0329/InternLM-learning/blob/main/%E4%B9%A6%E7%94%9F%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E7%AC%AC%E4%B8%80%E8%AF%BE.md

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/676289592

**Comment by xiyanjoy** [comment]
第一节课笔记：https://a0f4nqgrym.feishu.cn/wiki/T0BZwyqkji6hYakGbhmcpDcSnUc?from=from_copylink

**Comment by zzccchen** [comment]
第一节课笔记：https://github.com/zzccchen/notes_on_ml/blob/master/InternLm_tutorial/1.md

**Comment by godlikeS97** [comment]
第一节课笔记：https://blog.csdn.net/godlikesun97/article/details/135392853?spm=1001.2014.3001.5502

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/677111445
